script/create-dist.py

```diff
@@ -152,6 +152,10 @@ def create_symbols():
     dsyms = glob.glob(os.path.join(OUT_DIR, '*.dSYM'))
     for dsym in dsyms:
       shutil.copytree(dsym, os.path.join(DIST_DIR, os.path.basename(dsym)))
+  elif PLATFORM == 'win32':
+    pdbs = glob.glob(os.path.join(OUT_DIR, '*.pdb'))
```

Comment:
Nevermind, I hadn't done a full release build locally, only had a debug build in the `out/` folder on my Windows VM.